### ISSUE TITLE
Export to DOC?

### ISSUE THREAD

**Issue by Scribilicious** [notaapp/nota#105]
Possibility to export MD to Word Doc Format... maybe the possibility to import as well?

**Comment by astoilkov** [comment]
Hmm. We have considered this but it's very hard to do (technically). What we have been thinking is that this is something that can be done with an external tool and the user experience won't degrade too much. Yes, you would need to use another app but this would allow us to spend time on features that can't be done by an outside tool/app.

**Comment by Scribilicious** [comment]
Maybe using https://pandoc.org/ is a solution?

**Comment by astoilkov** [comment]
Yes. I've researched it a few times. It's a possible solution we should put more time researching it.

Have you considered installing pandoc yourself? Do you see problems with that?

**Comment by Scribilicious** [comment]
Yes I actually considered that. And then create a custom bash script that converts the md file to doc. However I found it easier for that purpose to use Marked 2. The solution doesn't work when using files from your iCloud... Check following Issue: https://github.com/notaapp/nota/issues/109

**Comment by astoilkov** [comment]
Aha. Ok. I will investigate the issue and get back to you.

Thanks for the additional information.
